databricks/koalas/frame.py

```diff
@@ -1893,6 +1893,9 @@ def index(self):
 
         Currently not supported when the DataFrame has no index.
 
+        ..Note:: In spark 2.3, this method only works when 'spark.sql.execution.arrow.enabled'
```

Comment:
`..Note::` -> `.. note::` - it affects documentation rendering.  spark -> Spark or PySpark